projects/admin/src/app/routes/issues-route.ts

```diff
@@ -71,7 +71,9 @@ export class IssuesRoute extends BaseRoute implements RouteInterface {
             exportFormats: [
               {
                 label: 'CSV',
-                format: 'csv'
+                format: 'csv',
+                endpoint: this._routeToolService.apiService.getEndpointByType('item/inventory'),
+                disableMaxRestResultsSize: true,
```

Comment:
This property seems not to be used

Reply:
this is used in PR on ng-core (https://github.com/rero/ng-core/pull/384).